azerothcore/azerothcore-wotlk
# DB/Quest Ulag the Cleaver

[Quest Ulag the Cleaver](https://vanilla-twinhead.twinstar.cz/?quest=1819)

##### CHANGES PROPOSED:

-  Move Ulag the Cleaver to SAI
-  


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1222


##### TESTS PERFORMED:

Tested in game with update. It's okay ( Players can now click on that ''Mausoleum Trigger'' and the doors are opening. The NPC "Ulag the cleaver" coming outside on Horse and attacking player) which is correct.


##### HOW TO TEST THE CHANGES:

1. .quest add 1819
2. .go xyz 2372.087646 363.687805 38.098766 0
3. Click on item in front of you '' Mausoleum Trigger ''
4. Take a look on the doors. 
5. They should opening and the '' Ulag the cleaver '' will come on horse attacking you.


##### KNOWN ISSUES AND TODO LIST:


- [x] Waypoint of the NPC is correct now
- [x] SAI is correct now


##### Target branch(es):

Master





Thanks @grimgravy
